src/command_modules/azure-cli-advisor/azure/cli/command_modules/advisor/_params.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
```

Comment:
Advice on not using this disable. Properly break the lines at 120 is recommended.